### ISSUE TITLE
Update Create Project buttons

### ISSUE THREAD

**Issue by ajefts** [appirio-tech/direct-app#370]
@skyhit Can you please update the buttons/links in Direct for creating projects to link to Connect?

This is the link:  https://connect.topcoder.com/new-project/

**Comment by skyhit** [comment]
@ajefts  you mean we will not create projects anymore directly in Direct?

**Comment by skyhit** [comment]
@ajefts this should be configurable for different environments, right? like for dev?

**Comment by ajefts** [comment]
Yes, we want to start moving towards Connect as the single place to start projects so we can establish some consistency and focus on Connect as the place to initiate projects, handle estimates and quotes, assign copilots, etc.

For the different envs, yes...there is connect.topcoder.com and connect.topcoder-dev.com

**Comment by skyhit** [comment]
@ajefts I see, I am on it

**Comment by skyhit** [comment]
@ajefts how should we deal with the button when creating challenge? just remove it?
![image](https://user-images.githubusercontent.com/58106/37381084-5624b908-2776-11e8-9c0c-2604fb3a8549.png)

**Comment by ajefts** [comment]
yes

**Comment by skyhit** [comment]
@ajefts if so, should we remove the Create New Project Wizard completely, like https://www.topcoder.com/direct/createNewProject.action

**Comment by skyhit** [comment]
@ajefts a F2F challenge to make the link tokenized is launched - https://www.topcoder.com/direct/contest/detail?projectId=30063584

**Comment by skyhit** [comment]
@ajefts PR created https://github.com/appirio-tech/direct-app/pull/373, let me know your thought.
